.woke.yaml

```diff
@@ -0,0 +1,8 @@
+ignore_files:
+  # Ignore ingress charm library as it uses non compliant terminology:
+  # whitelist.
+  - lib/charms/nginx_ingress_integrator/v0/ingress.py
+rules:
+  # Ignore "master" - the database relation event received from the library
+  # uses this terminology.
+  - name: master
```

Comment:
Do we need to include master in the whitelist?

Reply:
Yes due to postgres